app/src/main/java/com/gsandchase/mileagelogger/MainActivity.java

```diff
@@ -30,13 +30,10 @@
     TextView totalMileage;
     Button addMileageButton;
     EditText newMileage;
-    Context context;
 
-    private float weeklyMileage = 0, monthlyMileage = 0, annualMileage = 0;
```

Comment:
I want to remove as much state from the class as makes sense. The app contains an object that is responsible for keeping track of mileage totals, which replaces all of these data members.